.github/workflows/e2etest.yaml

```diff
@@ -43,5 +55,69 @@ jobs:
       adapter_name: kuma
       patternfile_name: deploy.yaml
       provider: Local
+      k8s_version: v1.20.1
+      sm_version: ${{ needs.SetPatternfile.outputs.sm_version }}
+      adapter_version:  ${{ needs.SetPatternfile.outputs.adapter_version }}
+      output_filename: data.json
     secrets:
-      token: ${{ secrets.PROVIDER_TOKEN }} 
\ No newline at end of file
+      token: ${{ secrets.PROVIDER_TOKEN }} 
+
+  UpdateDocs:
+    needs: TestKuma
+    if:  "always() && github.event_name != 'pull_request' "
+    runs-on: ubuntu-latest
+    steps:
+      - name: Skip if needed
+        run: |
+            echo "this is it ${{ needs.SkipIfNeeded.outputs.skipdocupdate }} "
+            if [ "${{github.event_name }}" == "push" ];then
+              echo "version=edge" >> $GITHUB_ENV
+            fi
+            if [ "${{github.event_name }}" == "release" ];then
+              echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+            fi
+      - uses: actions/checkout@master
+        with:
+          repository: meshery/meshery 
+          token: ${{ secrets.GH_ACCESS_TOKEN }}   
+      - name: DownloadJSON
+        uses: actions/download-artifact@v2
+        with:
+          name: data.json
+      - name: echo results
+        run: |
+          pretty=$(jq . data.json)
+          rm data.json
+          mkdir -p ./docs/_compatibility/meshery-kuma
+          cd ./docs/_compatibility/meshery-kuma
+          touch data.json
+          echo $pretty > data.json
+          cat data.json
+          touch test.md
+          filename=$(date)
+          filename="${filename// /_}" 
+          echo $filename
+          echo "---
+          timestamp: $(jq '.metadata.runon' data.json)
```

Comment:
@Revolyssup this format is causing Meshery Docs to break. @alphaX86 is working on a fix -https://github.com/meshery/meshery/pull/4820#issuecomment-1000349639